### DISCUSSION TITLE
Question from HK user

### DISCUSSION THREAD

**Discussion by steaminggoat** [cat-box/aws-foundry-ssl#38] [Q&A]
Thank you for providing this Job!
I am a DND player from Hong Kong, I would like to ask if the service has the possibility to support AWS in Japan, Hong Kong ?

**Comment by cat-box** [comment]
Hey, sorry I missed this :( and if it's still relevant, yes. Looks like the documentation accidentally got skipped over. The template was updated to allow deployment from any region. 

Hong Kong: ap-east-1
Japan: ap-northeast-1

The only caveat, these regions are untested so stack creation may fail if it doesn't support t2.micro instances. In that case, you will need to use t3.micro. Otherwise, you can send me a DM/message on reddit and I can take a look to verify.
